### PR TITLE
Header social nav

### PR DESCRIPTION
**Pull request by bengammon**
This will add markup, styles and logos for social sites in the header.

### CODE REVIEW

**Comment by peterkeating** [comment]
@bengammon Did we want to get this setup on the actual site as well? Wonder if including YouTube would be good too.

**Comment by bengammon** [comment]
Yes, adding YouTube is a good idea. I've started the process of getting this into the site with a branch for the banner widget. I'll extend the default list of logos in the recipe so the majority of the common ones are catered for out of the box.

File: content/Assets/Styles/components/nav/_social.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* ==========================================================================
+   #NAV/SOCIAL
+   ========================================================================== */
+
+.nav-social {
```
Comment (Newnab) [inline content/Assets/Styles/components/nav/_social.scss:5]: For BEM purposes I think this should be `.nav--social` (and same in the markup) - it's a modified version of an existing block (nav)

Comment (bengammon) [reply content/Assets/Styles/components/nav/_social.scss]: Not sure how I missed this. Especially as I'd have typed it out at least twice in separate files. Good spot - I've pushed changes.

**Review by Newnab — APPROVED** [review]

File: content/PatternLibrary/patterns/02-Components/nav-social/nav-social.hbs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<ul class="list list--horizontal nav--social">
+    <li class="nav__item">
+        <a href="#" aria-haspopup="true" aria-expanded="false"><img src="/content/PatternLibrary/logo-discord.svg" alt="Discord logo"></a>
```
Comment (Newnab) [inline content/PatternLibrary/patterns/02-Components/nav-social/nav-social.hbs:3]: I approved this before I realised, but - I think these aria tags are unnecessary. There's no popup or dropdown, they're direct links. Will address in BE implementation directly but you might want to update the PL too :)

**Pull request merged**